climada_petals/hazard/copernicus_forecast/handler.py

```diff
@@ -0,0 +1,560 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Module to handle seasonal forecast data from the Copernicus Climate Data Store (CDS)
+in the U-CLIMADAPT project.
+
+TODO: update prerequisites
+Prerequisites:
+1. CDS API client installation:
+   pip install cdsapi
+
+2. CDS account and API key:
+   Register at https://cds-beta.climate.copernicus.eu
+
+3. CDS API configuration:
+   Create a .cdsapirc file in your home directory with your API key and URL.
+   For instructions, visit:
+   https://cds-beta.climate.copernicus.eu/how-to-api#install-the-cds-api-client
+"""
+
+import os
```

Comment:
I would suggest to use Pathlib instead of os whenever possible.

Reply:
As you suggested, I replaced os with pathlib for improved path handling and readability.